pkg/cmd/cli/cmd/set/volume.go

```diff
@@ -237,11 +237,22 @@ func (v *VolumeOptions) Validate(args []string) error {
 
 func (a *AddVolumeOptions) Validate(isAddOp bool) error {
 	if isAddOp {
+		if len(a.MountPath) == 0 {
+			return errors.New("must provide --mount-path for --add operation")
```

Comment:
@fabianofranz I just added this.  turns out that I could add the volume without a `mount-path` and it did nothing, which annoyed me.